linux_os/guide/services/ntp/service_chronyd_or_ntpd_enabled/rule.yml

```diff
@@ -43,7 +43,7 @@ references:
     cis: 2.2.1.1
     cui: 3.3.7
     disa: "160"
-    nist: AU-8(1)
+    nist: AU-8(1),AU-12(1)
```

Comment:
```suggestion
    nist: CM-6(a),CM-6(b)
```

Reply:
Updating to CM is out of scope of this PR.